Gemfile.lock

```diff
@@ -1,6 +1,6 @@
 GIT
```

Comment:
@VaibhavUpreti we need to update the `mailkick` and it's dependent gems only!
@tachyons

Reply:
Sure, will resolve it by EOD!